client/zanata-client-commands/src/main/java/org/zanata/client/commands/PutUserOptionsImpl.java

```diff
@@ -33,14 +33,14 @@ public PutUserCommand initCommand() {
     }
 
     @Override
-    @Option(name = "--user-name", required = true,
+    @Option(name = "--user-name", required = false,
```

Comment:
`required` defaults to `false` for `@Option`. I think it would be better to omit it.